src/backend/efiling-api/jag-efiling-api.yaml

```
@@ -90,13 +90,13 @@ paths:
                 $ref: "#/components/schemas/UpdateDocumentResponse"
         "404":
           description: Submission not found
-  "/submission/{submissionId}":
+  "/submission/{submissionId}/config":
     parameters:
       - $ref: "#/components/parameters/submissionId"
       - $ref: "#/components/parameters/xTransactionId"
     get:
       summary: Get user detail for a given submission
```

this needs to be updated as this does not return user details for a submission anymore
```suggestion
      summary: Get submission configuration information
```